spec/lib/data_hygiene/data_sync_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe DataHygiene::DataSync do
+  let(:logger) { double(:logger, info: true, warn: true) }
+  let(:topics) do
+    [{ 'name' => 'Topic A', 'code' => 'TA' },{ 'name' => 'Topic B', 'code' => 'TB' }]
+  end
+  let(:gov_delivery_client) { double(:gov_delivery_client, fetch_topics: { 'topics' => topics }) }
+  subject { described_class.new(logger, 0) }
+
+  before do
+    allow(ENV).to receive(:[]).and_return('')
+    allow(ENV).to receive(:[]).with('ALLOW_GOVDELIVERY_SYNC').and_return('allow')
```

Comment:
It would be good to test the `validate!` method as well, to be sure that we can't run this in production.